src/main/java/gov/usgs/cida/pubs/domain/query/PagingFilter.java

```diff
@@ -0,0 +1,35 @@
+package gov.usgs.cida.pubs.domain.query;
+
+import javax.validation.constraints.Positive;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class PagingFilter {
```

Comment:
Would a better name be DeletedPagingFilter if this is only used for Deleted publications?